Test is very strict, barely fails on float32 using AOCL BLAS backend.
author: micketeer@gmail.com
diff -ru scipy.orig/linalg/tests/test_batch.py scipy/linalg/tests/test_batch.py
--- scipy.orig/linalg/tests/test_batch.py	2026-07-13 15:11:34.398555702 +0200
+++ scipy/linalg/tests/test_batch.py	2026-07-13 15:17:23.905657689 +0200
@@ -453,7 +453,7 @@
             x = x[..., np.newaxis]
             b = b[..., np.newaxis]
         assert_allclose(A @ x - b, 0, atol=2e-6)
-        assert_allclose(x, np.linalg.solve(A, b), atol=3e-6)
+        assert_allclose(x, np.linalg.solve(A, b), rtol=1e-6, atol=4e-6)
 
     @pytest.mark.parametrize('bdim', [(5,), (5, 4), (2, 3, 5, 4)])
     @pytest.mark.parametrize('dtype', floating)
@@ -467,7 +467,7 @@
             x = x[..., np.newaxis]
             b = b[..., np.newaxis]
         assert_allclose(A @ x - b, 0, atol=2e-6)
-        assert_allclose(x, np.linalg.solve(A, b), atol=3e-6)
+        assert_allclose(x, np.linalg.solve(A, b), rtol=1e-6, atol=4e-6)
 
     @pytest.mark.parametrize('l_and_u', [(1, 1), ([2, 1, 0], [0, 1 , 2])])
     @pytest.mark.parametrize('bdim', [(5,), (5, 4), (2, 3, 5, 4)])
